Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/CommonComponents/LocationDrawingsModal.tsx

```diff
@@ -0,0 +1,53 @@
+//******************************************************************************************************
+//  LocationDrawings.tsx - Gbtc
+//
+//  Copyright � 2023, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  11/06/2024 - Collins Self
+//       Generated original version of source code.
+//
+//******************************************************************************************************
+import React from 'react';
+import { Modal } from '@gpa-gemstone/react-interactive';
+import LocationDrawingsTable from '../Location/LocationDrawingsTable';
+
+interface IProps {
+    LocationID: number;
+    Show: boolean;
+    SetShow: (b: boolean) => void;
+}
+
+const LocationDrawingsModal = (props: IProps) => {
+    return (
+        <Modal
+            Show={props.Show}
+            Title={'Drawings'}
+            ShowX={true} Size={'lg'}
+            CallBack={() => props.SetShow(false)}
+            ShowCancel={false}
+            ConfirmText={'Done'}>
```

Comment:
Check with erika, She has a Confluence doc that specifies styling for modal buttons (text color, X etc)